aidial_adapter_bedrock/llm/model/claude/v3/adapter.py

```diff
@@ -78,6 +81,19 @@ async def chat(
             parsed_messages, self.storage
         )
 
+        tools = NOT_GIVEN
+        tools_mode = None
+        if params.tool_config is not None:
+            tools = [
+                to_claude_tool_config(tool_function)
+                for tool_function in params.tool_config.functions
+            ]
+            tools_mode = (
+                ToolsMode.NATIVE_TOOLS
+                if params.tool_config.is_tool
```

Comment:
We could replace `.is_tool` propery with `.tools_mode` which returns `ToolsMode`.
It only used in two places anyway.

```python
tools_mode = params.tools_mode
```

Reply:
Fixed